common/src/main/proto/TransportMessages.proto

```diff
@@ -138,7 +138,7 @@ message PbHeartbeatFromWorker {
   int32 fetchPort = 4;
   int32 replicatePort = 5;
   repeated PbDiskInfo disks = 6;
-  repeated string shuffleKeys = 7;
+  map<string, int64> shuffleDiskUsage = 7;
```

Comment:
This change will break the behavior of identifying expired shuffles in handleHeartbeatFromWorker.